modules/gsuite_enabled/main.tf

```diff
@@ -14,10 +14,25 @@
  * limitations under the License.
  */
 
-/*
-If we don't need to work with gsuite, we can use the project-factory without installing/configuring gsuite,
-it is necessary to delete this file that contains the gsuite resources.
-*/
+/******************************************
+  Locals configuration
+ *****************************************/
+
+locals {
+  project_number    = "${module.project-factory.project_number}"
+  api_s_account     = "${format("%s@cloudservices.gserviceaccount.com", local.project_number)}"
```

Comment:
Instead of re-calculating these outside the module, why not add them as outputs to the core module?

Reply:
definitely should have done this earlier!